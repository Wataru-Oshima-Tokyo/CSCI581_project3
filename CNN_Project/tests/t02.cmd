 input_1C.txt weights_1C.txt structure_1C.txt 0.01
