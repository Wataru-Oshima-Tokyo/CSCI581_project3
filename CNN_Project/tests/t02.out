0.2141650169574414 0.9999946851475886 0.9999998439360778 0.5451268671091033 
